src/pages/candidate-details/candidate-details.scss

```diff
@@ -4,19 +4,24 @@ page-candidate-details {
     height: 26px;
     float: right;
   }
-
-  .test-category-icon {
-    display: inline-block;
-    height: 25px;
-    width: auto;
-    margin-left: 8px;
-  }
-
   .test-start-time {
     float: right;
   }
-
   .previous-cancellation-title {
     margin-top: 8px;
   }
+  .card-content-ios{
+    padding: 0;
+  }
+  .grid{
+    padding: 0;
+  }
+  .mes-data-row{
+    padding: 22px 19px;
+  }
+  h2{
+    padding: 0 28px;
+    line-height: 42px;
```

Comment:
Need to consider user accessibility here, as line height needs to change with bigger font sizes. If line height could be left to inherit from the existing h2 class that would be preferable.

Reply:
![simulator screen shot - ipad pro 10 5-inch - 2019-03-06 at 11 39 22](https://user-images.githubusercontent.com/8069071/53878753-98da4900-4004-11e9-859a-b6f7b2d22bb6.png)
Does seem to look fine at largest size

Reply:
I believe the line height only really makes a difference if the text spans over multiple lines of text, so in the case of the header it may be redundant. As the default is 40px it may just be better to remove that line and keep the inherited one.